server.nix

```diff
@@ -12,9 +12,15 @@
     pkgs.dnsutils
     pkgs.htop
     pkgs.jq
-    pkgs.termite.terminfo
     pkgs.tmux
     pkgs.vim
+    # avoid compiling desktop stuff when doing cross nixos
+  ] ++ lib.optionals (pkgs.stdenv.hostPlatform == pkgs.stdenv.buildPlatform) [
+    # various terminfo packages
+    pkgs.termite.terminfo
+    pkgs.wezterm.terminfo
```

Comment:
FYI At least wezterm's terminfo is cross friendly with minimal dependencies https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/terminal-emulators/wezterm/default.nix#L123-L131 because if you ssh into a machine with wezterm as your terminal programs will be confused about the missing data.

Reply:
okay that one might be fine to move than.

Reply:
https://github.com/numtide/srvos/pull/23